### PR TITLE
chore(Activities): add front-end app skeleton

### PR DESCRIPTION
**Pull request by nicolaspayot**
Here's a front-end app skeleton for Activities domain. ES2015 with React, no CSS framework to date.

### CODE REVIEW

**Comment by ldez** [comment]
LGTM

**Comment by devcrafting** [comment]
I would add a little bit more tips for each dev tools...I did not know enough each and I would like to know which tools aims which goal. Thanks. Else it is good for me.

**Comment by nicolaspayot** [comment]
I updated the readme file with more infos on dev tools.

**Comment by devcrafting** [comment]
:+1:

File: Activities/WebApp/.babelrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    "react",
+    "es2015"
```
Comment (jeantil) [inline Activities/WebApp/.babelrc]: does es2106 exist ? if so shouldn't it be the choice here ?

**Comment by jeantil** [comment]
I just tried the `npm install` and I get various deprecation warnings 

```
npm install
npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm
npm WARN deprecated lodash@1.0.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0
npm WARN deprecated npmconf@2.1.1: this package has been reintegrated into npm and is now out of date with respect to npm
npm WARN prefer global node-gyp@3.2.1 should be installed with -g
```

then it pulls older versions 

```
Found PhantomJS at /usr/local/bin/phantomjs ...verifying
PhantomJS detected, but wrong version 2.1.1 @ /usr/local/bin/phantomjs.
Downloading https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-macosx.zip
Saving to /var/folders/gt/c0_8ryzj2m7fwmf07t68z0940000gn/T/phantomjs/phantomjs-1.9.8-macosx.zip
```

and then it complains about missing values 

```
npm WARN WebApp No repository field.
npm WARN WebApp No license field.
```

both repository and license could easily be filled. 

Overall, I am not so happy about the use of a scaffoldin tool, it immediately pulls what seems (to me) an insane amount of dependencies. `npm install` displays the dependency tree once it completes and that tree has  1092 lines for an empty project ! I am not an expert javascript developer but that sure looks like a lot of YAGNI to me. 
Are you sure all of this is really needed ?

stack :

```
$ npm -v
3.5.3
$ node -v
v5.5.0
$ sw_vers
ProductName:    Mac OS X
ProductVersion: 10.11.2
BuildVersion:   15C50
```

**Comment by nicolaspayot** [comment]
No problem for `repository` and `license` properties. I'll fix `phantomjs` version as well.

Some dependencies with npm are deprecated, that's right, but it's fine. This is just the way npm works. npm packages are "micro-module" based. For instance, a dependency like `webpack` will pull a lot (like A LOT) of dependencies but it's perfectly normal. Again, that's just the way it works.

Dependencies don't come from scaffolding but from tools that are necessary for modern web development process. The generator I chose is very light compared with others and I think this is the minimum for starting a React web app in good conditions.

**Comment by jeantil** [comment]
> This is just the way npm works. npm packages are "micro-module" based.

I am not challenging that

> The generator I chose is very light compared with others and I think this is the minimum for starting a React web app in good conditions.

I'll trust you on that, I couldn't challenge it if I wanted to :)
I was alerted by the reference to a yeoman template and the amount of dependencies pulled and wanted to make sure nothing useless was being pulled. 

> No problem for `repository` and `license` properties. I'll fix `phantomjs` version as well.

Once that's in the PR will LGTM :)

**Comment by nicolaspayot** [comment]
- `repository` and `license` properties added
- `phantomjs` dependencies fixed

**Pull request merged**